app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
Oh it's not a big deal Jeff! Thanks for your time and response on this. Every new feature takes a lot of discussion and thoughts before adaptation. And it is best to discuss it with the actual changes in the code. I am leaving this PR open in case you want to discuss it further.